core/src/main/java/com/novoda/noplayer/Player.java

```diff
@@ -101,13 +102,13 @@
     void detach(PlayerView playerView);
 
     /**
-     * Retrieves all of the available {@link PlayerAudioTrack} of a prepared Player.
+     * Retrieves all of the available {@link PlayerAudioTrack} of a prepared Player as a first class collection.
      *
-     * @return A list of available {@link PlayerAudioTrack}.
+     * @return {@link AudioTracks} that contains a list of available {@link PlayerAudioTrack}.
      * @throws IllegalStateException - if called before {@link Player#loadVideo(Uri, ContentType)}.
      * @see Player.PreparedListener
      */
-    List<PlayerAudioTrack> getAudioTracks() throws IllegalStateException;
+    AudioTracks getAudioTracks() throws IllegalStateException;
```

Comment:
`PlayerAudioTracks getPlayerAudioTracks()`

I mean Ideally `PlayerAudioTrack` -> `AudioTrack` 😁 but I guess not possible